### PR TITLE
Tmedia 741 no content

### PR DESCRIPTION
**Pull request by malavikakoppula**
## Description

https://arcpublishing.atlassian.net/browse/TMEDIA-741?focusedCommentId=787240

## Jira Ticket

- [TMEDIA-741](https://arcpublishing.atlassian.net/browse/TMEDIA-741)

## Test Steps

1. Checkout this branch `git checkout TMEDIA-741-no-content
2. Run fusion repo with linked blocks npx fusion start -f -l @wpmedia/large-promo-block
3. Open https://localhost/pagebuilder/pages and open Reference - All Blocks
4. Configure Large Promo Block to use _id : 7TLCY4UZN5F4BA5I4FP62E6NYE

### Before

Screen Shot 2022-08-02 at 1 05 46 PM

### After

Screen Shot 2022-08-02 at 1 04 59 PM

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/large-promo-block/features/large-promo/default.jsx [hunk 1]
```diff
@@ -33,18 +34,18 @@ const BLOCK_CLASS_NAME = "b-large-promo";
...
+		aspectRatio,
+		imageOrVideoLabelText,
 		imageOverrideURL,
 		playVideoInPlace,
 		showByline,
 		showDate,
 		showDescription,
 		showHeadline,
 		showImage,
+		showImageOrVideoLabel,
```
Comment (nschubach) [inline blocks/large-promo-block/features/large-promo/default.jsx]: Thanks!

File: blocks/large-promo-block/features/large-promo/default.jsx [hunk 2]
```diff
@@ -188,28 +191,37 @@ const LargePromoItem = ({ customFields, arcSite }) => {
...
 									aspectRatio={aspectRatio}
-									embedMarkup={customFields?.content?.embed_html}
+									embedMarkup={embedMarkup}
 									viewportPercentage={viewportPercentage}
 								/>
 							) : (
 								<Image
 									alt={content?.headlines?.basic || null}
-									src={targetImage}
+									src={targetImage || fallbackImage}
```
Comment (nschubach) [inline blocks/large-promo-block/features/large-promo/default.jsx]: VERY minor change... can we pull this up to line 159:

```
const promoImage = imageOverrideURL || getImageFromANS(content) || fallbackImage;
```
```suggestion
									src={promoImage}
```

File: blocks/large-promo-block/features/large-promo/default.jsx [hunk 3]
```diff
@@ -225,9 +237,7 @@ const LargePromoItem = ({ customFields, arcSite }) => {
 					</MediaItem>
 				) : null}
 				<Stack className={`${BLOCK_CLASS_NAME}__text`}>
-					{showOverline && (url || text) ? (
-						<Overline href={url ? formatURL(url) : null}>{text}</Overline>
-					) : null}
+					{showOverline ? <Overline href={url}>{text}</Overline> : null}
```
Comment (nschubach) [inline blocks/large-promo-block/features/large-promo/default.jsx]: Was this lost or not necessary?
```suggestion
					{showOverline ? <Overline href={formatURL(url)}>{text}</Overline> : null}
```

Comment (malavikakoppula) [reply blocks/large-promo-block/features/large-promo/default.jsx]: @nschubach Thanks, removed formatURL() in [3b044d8](https://github.com/WPMedia/arc-themes-blocks/pull/1462/commits/3b044d874ca3e8201b6c45351ab8f9d750f61534)

File: blocks/large-promo-block/features/large-promo/default.jsx (hunk 2 above)
Comment (malavikakoppula) [reply blocks/large-promo-block/features/large-promo/default.jsx]: @nschubach Thanks, added fallbackimage to promoImage[71b773e](https://github.com/WPMedia/arc-themes-blocks/pull/1462/commits/71b773e11f78a8f2ba8ecd4d58764d3ac2ef3a42)

**Comment by nschubach** [comment]
Just curious if this is expected (the `max-height` is causing the gap to increase):
image
If I remove a lot of these styles, it seems to resolve a couple issues (re: height)
image
image

**Comment by malavikakoppula** [comment]
> 

removing max-height is causing image height to me more than expected:
Screen Shot 2022-08-03 at 1 13 23 PM

File: blocks/large-promo-block/features/large-promo/default.jsx (hunk 3 above)
Comment (nschubach) [reply blocks/large-promo-block/features/large-promo/default.jsx]: @malavika37  be sure to add this one back in.

Comment (malavikakoppula) [reply blocks/large-promo-block/features/large-promo/default.jsx]: @nschubach added it back in [beaaa9c](https://github.com/WPMedia/arc-themes-blocks/pull/1462/commits/beaaa9ce26ab3304976068d9261376934f2b47b5)

Comment (nschubach) [reply blocks/large-promo-block/features/large-promo/default.jsx]: @malavika37 not this specific case.

**Comment by matthewroach** [comment]
The image grow as shown in your comment @malavikakoppula is fine - the width and height of the image shouldn't be controlled via CSS `max-width` we should be using image resizer

**Review by nschubach — CHANGES_REQUESTED** [review]
We just need to add in that one `formatUrl()` case and this should be gold.

**Review by nschubach — DISMISSED** [review]
Nevermind my last review the change was in the PR that I just merged in.

**Comment by JackHowa** [comment]
@malavikakoppula @nschubach I updated this to ensure that the gallery and video label removal was not continued.

File: blocks/large-promo-block/features/large-promo/default.test.jsx [hunk 4]
```diff
@@ -288,9 +292,9 @@ describe("Large Promo", () => {
 		expect(screen.queryByText(largePromoMock.headlines.basic)).not.toBeNull();
 		expect(screen.queryByText(largePromoMock.description.basic)).toBeNull();
 		expect(screen.queryByText("December 02, 2019 at 6:58 pm UTC")).toBeNull();
-		expect(screen.queryByRole("img")).not.toBeNull();
 	});
 
+	// image icon not working as expected
```
Comment (JackHowa) [inline blocks/large-promo-block/features/large-promo/default.test.jsx]: oh damn, I think I added this? It can be removed

Comment (JackHowa) [reply blocks/large-promo-block/features/large-promo/default.test.jsx]:
```suggestion
```

**Comment by JackHowa** [comment]
@malavikakoppula can you make a feature pack pr? Looks like a couple styles changed

**Comment by JackHowa** [comment]
Sorry, just trying to catch up on the progress on this ticket. I know it's difficult to keep track of all of the stories.

Screen Shot 2022-08-09 at 15 56 47

looks like a previous story had text on this one. That was not correct, right? it looks like it was accepted before. 

https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=62f2c86ef802434e8a05a707

https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=62f2c86ef802434e8a05a708

**Comment by malavikakoppula** [comment]
> https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=62f2c86ef802434e8a05a707

For the last three stories there is no text displaying according to V1.

**Comment by malavikakoppula** [comment]
> @malavikakoppula can you make a feature pack pr? Looks like a couple styles changed

Yes, jack. Created a new PR on feature-pack repo.

**Review by ecarlisle — DISMISSED** [review]
@malavikakoppula : I am seeing the two cases that Tariq was asking about:

1. A fallback image is used when the block is initially selected.
2. There is the "Play video in place" option.

I am still seeing the following warnings on a locally-published page:
image
My assumptions are that:
1. My local Feature Pack isn't seeing the `gallery-text` or `video-text` because it's not seeing the changes you've made in this PR - moving those 2 phrases from Shared Styles to Global.
2. V2 image resizer isn't in the scope of the ticket, so the expectation for `auth` can be disregarded for now.

I'm approving the PR and setting the tag to "Ready for Design QA."

**Comment by nschubach** [comment]
> @malavikakoppula : I am seeing the two cases that Tariq was asking about:
> 
> 1. A fallback image is used when the block is initially selected.
> 2. There is the "Play video in place" option.
> 
> I am still seeing the following warnings on a locally-published page: image My assumptions are that:
> 
> 1. My local Feature Pack isn't seeing the `gallery-text` or `video-text` because it's not seeing the changes you've made in this PR - moving those 2 phrases from Shared Styles to Global.
> 2. V2 image resizer isn't in the scope of the ticket, so the expectation for `auth` can be disregarded for now.
> 
> I'm approving the PR and setting the tag to "Ready for Design QA."

It seems that `global-phrases-block` needs to be updated for this story as well.

**Review by ecarlisle — APPROVED** [review]
I've confirmed that:
1. Missing global phrases is no longer being reported as a warning.
2. The "no auth" is an expected behavior of the `Image` component.
3. Issues reported by QA are resolved.
4. Block functionality is per requirements.

Great job, @malavikakoppula !

**Comment by nschubach** [comment]
@haganka : This is pre-header/line token work.  We'll need to open a story to adjust these to the new tokens.  I don't want to hold this up for that change.

**Review by nschubach — APPROVED** [review]

**Pull request merged**